sdk/docs/core/README.md

```diff
@@ -72,7 +72,7 @@ There are many functions to manipulate `az_span` instances. You can slice (subse
 
 ### Strings
 
-A string is a span of UTF-8 characters. It's not a zero-terminated string. Defined in [inc/az_span.h](inc/az_span.h).
+A string is a span of UTF-8 characters. It's not a zero-terminated string. Defined in [inc/az_span.h](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/inc/az_span.h).
```

Comment:
Given we already use absolute paths in this file, this change makes sense.

However, it is the opposite of making the links relative so they work on any branch (that @danewalton's did in https://github.com/Azure/azure-sdk-for-c/pull/853).

**Question:** Do we want to keep the full/absolute path or go with relative paths?

Something like:
```suggestion
A string is a span of UTF-8 characters. It's not a zero-terminated string. Defined in [inc/az_span.h](../../inc/azure/core/az_span.h).
```